spec/soapy_bing/ads/bulk/parsers/csv_parser_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require 'json'
```

Comment:
> I like the requires in files that actually uses the lib. 

I kinda, too, at least in theory. The problem is that ruby loads everything into the global namespace, across all files. 

So if you `require 'csv'` in `spec/foo_spec.rb`, then use `CSV.parse` in `lib/foo.rb`, but forgot to add `require 'csv'` to `lib/foo.rb`, the specs will be green but it won't work in the actual application. 

Same issue if you accidentally use `CSV.parse` in `lib/foo.rb`, but only required it in `lib/bar.rb`. That effect is amplified more when using rails, because it autoloads a lot of common libraries anyway.

Because of that, I think it's good practise to just put all library-requires in the top-level code file. WDYT?

> Also, I just followed the convention that was already used in the project. If we decided to change, could be done in a different PR.

Fair enough, let's keep it for now then.